### PR TITLE
Update Test Coverage

### PR DESCRIPTION
**Pull request by lsyurea**
More Testing on Main, App Parameters, MainApp and storage.

### CODE REVIEW

File: src/test/java/seedu/address/storage/JsonAdaptedCommentTest.java [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    @Test
+    public void constructor_null_throwsNullPointerException() {
+        assertThrows(NullPointerException.class, () -> new Comment(null));
+    }
+
+    @Test
+    public void constructor_invalidCommentName_throwsIllegalArgumentException() {
+        String invalidCommentName = "";
+        assertThrows(IllegalArgumentException.class, () -> new Comment(invalidCommentName));
+    }
```
Comment (Choonyan02) [inline src/test/java/seedu/address/storage/JsonAdaptedCommentTest.java:34]: I like that you included edge cases, such as null or empty inputs.

File: src/test/java/seedu/address/storage/JsonAdaptedCommentTest.java [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.commons.exceptions.IllegalValueException;
+import seedu.address.model.fields.Comment;
+
+public class JsonAdaptedCommentTest {
```
Comment (Choonyan02) [inline src/test/java/seedu/address/storage/JsonAdaptedCommentTest.java:11]: You covered different aspects of JsonAdaptedComment class, including constructors, conversion methods, and exception handling. Well done!

**Review by Choonyan02 — APPROVED** [review]
This pull request appears to be in great shape, you followed the coding standards. I appreciate that you helped change some of the test methods name to make it consistent too! I tried out the new DarkTheme, and it looks good. Overall LGTM!
